docs/components/custom_components.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Custom Components
+parent: Components
+permalink: /docs/Custom_Components
+---
+
+# Implementing a Custom Component
+
+The components we provide here may not fit your specific needs, and you'll need to write a custom component. Below is an example of how this could be done using React.
```

Comment:
here's a suggestion to avoid "we"
The components that are provided here might not fit your specific needs, and you'll need to write a custom component. 

and we try to avoid sensory words that assume sighted users. so, i try to replace terms like above and below whenever i see them.
The following example show you how to write a custom component by using React.